how-to/workspace-platform-starter/client/src/framework/favorite.ts

```diff
@@ -57,6 +62,22 @@ export async function init(
 	} else {
 		logger.warn("Options were not passed to the favorite provider so it cannot be initialized.");
 	}
+
+	conditionsProvider.registerCondition("favorites", async () => options?.enabled ?? false);
+
+	for (const type of [
+		FAVORITE_TYPE_NAME_APP,
+		FAVORITE_TYPE_NAME_WORKSPACE,
+		FAVORITE_TYPE_NAME_PAGE,
+		FAVORITE_TYPE_NAME_QUERY
+	]) {
+		conditionsProvider.registerCondition(
+			`favorites-${type}`,
+			async () =>
+				(options?.enabled ?? false) &&
+				(options?.supportedFavoriteTypes ?? []).includes(type as FavoriteTypeNames)
+		);
+	}
 }
```

Comment:
Is there a condition just for favorites? i.e. these are ..I want to show this if apps favorites is there but what if you wanted the module to worry about that e.g. it shows apps and workspace favorites

Reply:
line 66